cpp/include/Ice/StreamHelpers.h

```diff
@@ -13,6 +13,10 @@
 #include <optional>
 #include <string_view>
 
+#if __has_include(<span>)
```

Comment:
That's the preferred and portable way to conditionally include span. Span is a c++20 and up feature.